python/pyspark/worker.py

```diff
@@ -150,7 +150,8 @@ def read_udfs(pickleSer, infile, eval_type):
 
     if eval_type == PythonEvalType.SQL_PANDAS_SCALAR_UDF \
        or eval_type == PythonEvalType.SQL_PANDAS_GROUP_MAP_UDF:
-        ser = ArrowStreamPandasSerializer()
+        timezone = utf8_deserializer.loads(infile)
```

Comment:
who will read this timezone?

Reply:
The timezone will be read by the following `ArrowStreamPandasSerializer`.